TM1py/Objects/MDXView.py

```diff
@@ -15,13 +15,39 @@ class MDXView(View):
         IMPORTANT. MDXViews can't be seen through the old TM1 clients (Archict, Perspectives). They do exist though!
     """
 
-    def __init__(self, cube_name: str, view_name: str, MDX: str):
+    def __init__(self, cube_name: str, view_name: str, MDX: str, Meta: dict = {}):
```

Comment:
Please name it `meta` instead of `Meta`. 

Unfortunately, we broke with the Python naming conventions many years ago when we named the other argument `MDX` instead of `mdx`, but let's not introduce any more unconventional names